### ISSUE TITLE
ppc64: `systemd-network-generator.service` error - Failed to create temporary unit file in '/run/systemd/network': Permission denied

### ISSUE THREAD

**Issue by HuijingHei** [coreos/fedora-coreos-tracker#1587]
### Describe the bug

After installation FCOS 39.20230916.1.2 on baremental with ppc64le, get failed `systemd-network-generator.service`.

Not sure there is any side effect for this, the network is up and can ssh to the host.

### Reproduction steps

Install FCOS 39.20230916.1.2 on baremental with ppc64le

### Expected behavior

`systemd-network-generator.service` should work well.

### Actual behavior

```
[systemd]
Failed Units: 1
  systemd-network-generator.service

[core@ibm-ic922-07 ~]$ systemctl status systemd-network-generator.service
× systemd-network-generator.service - Generate network units from Kernel command line
     Loaded: loaded (/usr/lib/systemd/system/systemd-network-generator.service; enabled; preset: enabled)
    Drop-In: /usr/lib/systemd/system/service.d
             └─10-timeout-abort.conf
     Active: failed (Result: exit-code) since Fri 2023-09-29 12:47:31 UTC; 9min ago
       Docs: man:systemd-network-generator.service(8)
    Process: 3343 ExecStart=/usr/lib/systemd/systemd-network-generator (code=exited, status=1/FAILURE)
   Main PID: 3343 (code=exited, status=1/FAILURE)
        CPU: 6ms

[core@ibm-ic922-07 ~]$ journalctl | grep systemd-network-generator
Sep 29 12:47:31 localhost.localdomain systemd[1]: Starting systemd-network-generator.service - Generate network units from Kernel command line...
Sep 29 12:47:31 localhost.localdomain systemd[1]: systemd-network-generator.service: Main process exited, code=exited, status=1/FAILURE
Sep 29 12:47:31 localhost.localdomain systemd[1]: systemd-network-generator.service: Failed with result 'exit-code'.
Sep 29 12:47:31 localhost.localdomain systemd[1]: Failed to start systemd-network-generator.service - Generate network units from Kernel command line.
Sep 29 12:47:31 localhost.localdomain systemd-network-generator[3343]: Failed to create temporary unit file in '/run/systemd/network': Permission denied
```

### System details

FCOS 39.20230916.1.2

**Comment by dustymabe** [comment]
Thanks for reporting the issue. Can you tell me what the kernel command line is on the machine?

I think this is a duplicate of https://github.com/coreos/fedora-coreos-tracker/issues/1542 and should be fixed in the next `next` release. Should already be fixed in `next-devel` if you want to check that.

**Comment by HuijingHei** [comment]
Thanks @dustymabe , verify that the issue is duplicate of https://github.com/coreos/fedora-coreos-tracker/issues/1542, and fixed on `next-devel`: `FCOS 39.20231002.10.0` with `selinux-policy-38.28-1.fc39.noarch`.

```
Fedora CoreOS 39.20231002.10.0
[core@ibm-ic922-03 ~]$ systemctl status systemd-network-generator.service
● systemd-network-generator.service - Generate network units from Kernel comman>
     Loaded: loaded (/usr/lib/systemd/system/systemd-network-generator.service;>
    Drop-In: /usr/lib/systemd/system/service.d
             └─10-timeout-abort.conf
     Active: active (exited) since Thu 2023-10-05 03:10:20 UTC; 5min ago
       Docs: man:systemd-network-generator.service(8)
    Process: 3433 ExecStart=/usr/lib/systemd/systemd-network-generator (code=ex>
   Main PID: 3433 (code=exited, status=0/SUCCESS)
        CPU: 7ms

[core@ibm-ic922-03 ~]$ rpm -q selinux-policy systemd
selinux-policy-38.28-1.fc39.noarch
systemd-254.1-2.fc39.ppc64le
```

**Comment by HuijingHei** [comment]
Verify the new next `Fedora CoreOS 39.20231002.1.1` includes fixed `selinux-policy-38.28-1.fc39.noarch`, close the issue now.
